README.md

```diff
@@ -13,6 +13,12 @@ In this project, we're required to create a Tic Tac Toe Game using Ruby with OOP
 
 > The main goal of this project is to ensure the mastery of the methods ruby provides and to learn how to replicate them using basic coding knowledge.
 
+
+### In order to run the test cases for the methods, Please follow these steps
+
+- `gem install rspec`
+- Run `rspec` to see your testcases whether they are passed or failed
+
```

Comment:
## More detailed info 
- [ ] Add a very detailed information on how about how to run your application. Maybe we can because we used to it, but how about other people who just ran out with your project.

Reply:
Hi @bertil291utn we modified read me file as requested.